_posts/2016-03-29-the-landing-page-mvp.markdown

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: "The Landing Page MVP"
+date: 2016-03-29 11:03:00
+categories: ["lean-startup", "mvp"]
+author: "etagwerker"
+---
+
+There is no good reason why an **MVP** should take more than one month. If that happens, it means that the scope of the [minimum viable product](https://leanstack.com/minimum-viable-product/) wasn't small enough.
+
+You want to build the smallest feature set in order to start learning from your target market. It doesn't have to be feature complete. It doesn't even have to offer a feature. It doesn't even need to be a web-based MVP.
+
+<!--more-->
+
+## Lean Canvas
+
+If you haven't already, write the first draft of your [Lean Canvas](https://leanstack.com/lean-canvas/). It's a good exercise to define the **problem**, **solution**, **customer segments**, **unique value proposition**, cost structure and revenue streams.
+
+Keep in mind that each post-it in your Lean Canvas is a hypothesis. **It's what you consider as the problem, not necessarily what the market considers as the problem**.
+
+The goal here is to identify your riskiest hypothesis.
+
+### Customer Segments
+
+Ask yourself: Who is your ideal client?
+
+If the answer is vague, try again. Do they have a particular profession? What do they like to do? How old are they? Are they married/single? How many customers does your business model have?
+
+I like to use this form to define your target market:
+
+* As a loyal customer to my local coffee shop, I'd like to get rewards for being loyal to my shop.
+* As a local small business owner, I want to offer my repeat customers rewards so that they continue to shop at my store.
+
+
+### Problem
+
+Now that you've defined the customer segments hypotheses, you can think about the problem. What problem do they have? What's their pain?
+
+Once you have defined problem and customer segments, you can create your landing page MVP.
+
+## The Landing Page MVP
+
+Sometimes an MVP can be as simple as a landing page. If that's the case, you can use one of these services:
+
+* [Unbounce](http://unbounce.com)
+* [Lead Pages](http://www.leadpages.net)
+
+You don't need a programmer or software boutique for this. You can build a landing page yourself and drive people to it.
+
+### Problem Description
+
+The most important part of the landing page is the problem description. You want the people in your target market to read the problem description and relate to it.
+
+You will find these type of people in your target market:
```

Comment:
It should be `You will find these types of people`, but it still sounds strange to me, maybe it could be rephrased?